conu/backend/podman/utils.py

```diff
@@ -0,0 +1,28 @@
+def check_docker_command_works():
```

Comment:
The method should be called `check_podman_command_works()`. Also functionality is still for docker.